blockchain/blockchain-wallet-v4-frontend
# Add link to resend email verification code

## Description
Adds a link to resend the email verification code on the verification entry page

## Change Type
Feature

## Testing Steps
Security center => verify email => send code => verify resend code link appears and works.

## PR Creator Checklist

## PR Reviewer Checklist
